webapp/src/utils/date_utils.js

```diff
@@ -1,49 +0,0 @@
-export function formatDate(date, useMilitaryTime = false) {
```

Comment:
Interesting. This function was indeed not used anywhere. Thanks for removing this dead code :+1: